cmd/block-generator/docker/Dockerfile-generator

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13.12-alpine
+FROM golang:1.16.11-alpine
```

Comment:
perhaps in docker we can use the newest version of go?

Reply:
i'm all for moving to 1.17. We need to get managers approval on this. Can you guys talk to Tsachi and see if he approves going straight to 1.17 everywhere?

Reply:
> i'm all for moving to 1.17. We need to get managers approval on this. Can you guys talk to Tsachi and see if he approves going straight to 1.17 everywhere?

@tsachiherman